pkg/cmd/server/api/v1/types.go

```diff
@@ -35,6 +37,13 @@ type NodeConfig struct {
 
 	// AllowDisabledDocker if true, the Kubelet will ignore errors from Docker.  This means that a node can start on a machine that doesn't have docker started.
 	AllowDisabledDocker bool `json:"allowDisabledDocker"`
+
+	// PodManifestPath specifies the path for the pod manifest file(s)
+	// The path could point to a single file or a directory that contains multiple manifest files
+	// This is used by the Kubelet to create pods on the node
+	PodManifestPath string `json:"podManifestPath"`
+	// FileCheckFrequency is the duration between checking manifest files for new data
+	FileCheckFrequency time.Duration `json:"fileCheckFrequency"`
```

Comment:
can these be grouped, or named so it is obvious that `fileCheckFrequency` refers to the `podManifestPath`? In the serialized file, stuff gets written alphabetically